src/server/services/afis/afis-facturen.ts

```diff
@@ -260,15 +260,43 @@ function transformFacturen(
 ): AfisFacturenResponse {
   const feedProperties = getFeedEntryProperties(responseData);
   const count = responseData?.feed?.count ?? feedProperties.length;
-  const facturenTransformed = feedProperties.map((invoiceProperties) => {
-    return transformFactuur(invoiceProperties, sessionID, deelbetalingen);
-  });
+  const facturenTransformed = feedProperties
+    .filter((invoiceProperties) => {
+      return shouldFilterOutFactuur(invoiceProperties.PostingDate);
+    })
+    .map((invoiceProperties) => {
+      return transformFactuur(invoiceProperties, sessionID, deelbetalingen);
+    });
   return {
     count,
     facturen: facturenTransformed,
   };
 }
 
+function shouldFilterOutFactuur(postingDate: string): boolean {
+  const postingDateAsDate = new Date(postingDate);
+  const now = new Date();
```

Comment:
```suggestion
  const FACTUREN_DOWNLOADABLE_AFTER_TODAY_O_CLOCK = 19;
  const dateTodayAvailableAfterOClock = new Date();
  dateTodayAvailableAfterOClock.setHours(
    FACTUREN_DOWNLOADABLE_AFTER_TODAY_O_CLOCK,
    0,
    0
  );
```

Reply:
Ik vind dit heel moeilijk te begrijpen `now` is bekend en kort, en de `19` is letterlijk `sevenOClock`.

Reply:
Ik denk dat je 2 dingen doorelkaar haalt.

`const FACTUREN_DOWNLOADABLE_AFTER_TODAY_O_CLOCK = 19;` // Is een beschrijvende naam en zegt wat over de aard van de waarde => Facturen zijn te downloaden na vandaag om 19:00

```
const dateTodayAvailableAfterOClock = new Date();
dateTodayAvailableAfterOClock.setHours(
    FACTUREN_DOWNLOADABLE_AFTER_TODAY_O_CLOCK,
    0,
    0
  );
 ```
 
 Is de datum van vandaag om 19:00 daarmee kun je makkelijk vergelijken toch?

Maar, die wijziging hoeft alleen als je iets doet als ik hier beneden heb geschreven:


```
const isPastDate = !isToday(datePosted) && isPast(datePosted);
  const isTodayAfterDownloadOclock =
    isToday(datePosted) && isAfter(datePosted, dateTodayAvailableAfterOClock);

  return isPastDate || isTodayAfterDownloadOclock;
```

Reply:
Ah ja ik snap wat je bedoelt, bedankt voor het uitleggen. Ik heb die naam veranderd